includes/ModuleController.php

```diff
@@ -10,89 +10,88 @@
 
 /**
  * Class ModuleController
- * 
```

Comment:
Changes in `includes/ModuleController.php` other than the formatting are
- Switch from `date` to `gmdate`, phpcs found a code standards violation `date() is affected by runtime timezone changes which can cause date/time to be| incorrectly displayed. Use gmdate() instead.`
- Use the `is_ecommerce_plan` function